who_l3_smart_tools/utils/__init__.py

```diff
@@ -1,15 +1,4 @@
-import re
-
-split_re = re.compile(r"[\W_]")
-
-
-def camel_case(s: str) -> str:
-    if s is None:
-        return ""
-
-    return "".join(
-        [
-            s.lower() if i == 0 else s.capitalize()
-            for i, s in enumerate(split_re.split(s))
-        ]
-    )
+# ruff: noqa: F403
+from .counter import *
+from .cql_helpers import *
+from .str import *
```

Comment:
These can become absolute imports since we decided to go with that approach.

Reply:
Yeah, either way works. I sort of figured this is the one case where relative imports are actually less confusing about what's happening, but happy to change.